genai/tools/tools_code_exec_with_txt_local_img.go

```diff
@@ -0,0 +1,111 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package tools shows examples of various tools that Gemini model can use to generate text.
+package tools
+
+// [START googlegenaisdk_tools_code_exec_with_txt_local_img]
+import (
+	"context"
+	"fmt"
+	"io"
+	"os"
+
+	genai "google.golang.org/genai"
+)
+
+// generateWithCodeExecAndImg shows how to generate text using the code execution tool and a local image.
+func generateWithCodeExecAndImg(w io.Writer) error {
+	ctx := context.Background()
+
+	client, err := genai.NewClient(ctx, &genai.ClientConfig{
+		HTTPOptions: genai.HTTPOptions{APIVersion: "v1"},
+	})
+	if err != nil {
+		return fmt.Errorf("failed to create genai client: %w", err)
+	}
+
+	// Image source:
+	//   https://upload.wikimedia.org/wikipedia/commons/thumb/3/3f/Monty_open_door.svg/640px-Monty_open_door.svg.png
+	imgBytes, err := os.ReadFile("./testdata/640px-Monty_open_door.svg.png")
```

Comment:
@CadillacBurgess1  - can you please re-check if this file is available or not ?

I see errors `failed to read input file: open ./testdata/640px-Monty_open_door.svg.png: no such file or directory`